various minor changes to staking pages (#8692)
ethereum/ethereum-org-website: src/content/staking/saas/index.md

## What to consider {#what-to-consider}

> There are a growing number of SaaS providers to help you stake your ETH, but each come with different risks and benefits. Note that **all** SaaS options require additional trust assumptions compared to home-staking. There is additional code wrapping the Ethereum clients that might not be open and auditable. SaaS also potentially has a detrimental effect on network decentralization. Depending on the setup, you may not control your own validator - the operator _could_ act dishonestly using your ETH.

Reviewer: ```suggestion
There are a growing number of SaaS providers to help you stake your ETH, but each comes with different benefits and risks. You should consider that all SaaS options require additional trust assumptions compared to home-staking. Saas options may have additional code wrapping the Ethereum clients that is not open or auditable. SaaS also has a detrimental effect on network decentralization. Depending on the setup, you may not control your validator - the operator could act dishonestly using your ETH.
```

> There are a growing number of staking-as-a-service providers to help you stake your ETH, but each come with different risks and benefits. Note that **all** SaaS options require additional trust assumptions compared to home-staking. There is additional code wrapping the Ethereum clients that might not be open and auditable. SaaS also potentially has a detrimental effect on network decentralization. Depending on the setup, you may not control your own validator - the operator _could_ act dishonestly using your ETH.

Reviewer: ```suggestion
There are a growing number of SaaS providers to help you stake your ETH, but each come with different risks and benefits. Note that **all** SaaS options require additional trust assumptions compared to home-staking. There is additional code wrapping the Ethereum clients that might not be open and auditable. SaaS also potentially has a detrimental effect on network decentralization. Depending on the setup, you may not control your own validator - the operator _could_ act dishonestly using your ETH.
```
